src/context/LogContext/index.tsx

```diff
@@ -245,24 +268,35 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
         cache.clear(index, 0);
       },
       scrollToLine,
+
       setCaseSensitive: (caseSensitive: boolean) => {
         dispatch({ type: "SET_CASE_SENSITIVE", caseSensitive });
+        Cookie.set(CASE_SENSITIVE, caseSensitive.toString(), { expires: 365 });
       },
+      setExpandableRows,
       setFileName: (fileName: string) => {
         dispatch({ type: "SET_FILE_NAME", fileName });
       },
+      setFilterLogic,
       setLogMetadata: (logMetadata: LogMetadata) => {
         dispatch({ type: "SET_LOG_METADATA", logMetadata });
       },
       setSearch: (searchTerm: string) => {
         dispatch({ type: "SET_SEARCH_TERM", searchTerm });
       },
+      setWrap,
       togglePrettyPrint: (v: boolean) => {
         setPrettyPrint(v);
         Cookie.set(PRETTY_PRINT_BOOKMARKS, v.toString(), { expires: 365 });
       },
     }),
     [
+      wrap,
+      setWrap,
+      filterLogic,
+      setFilterLogic,
+      expandableRows,
+      setExpandableRows,
```

Comment:
How do you feel about grouping these into a `preferences` object?

Reply:
good idea :]